nix/shells/pkg-sets/rust.nix

```diff
@@ -12,7 +11,6 @@
     REPORTER_CONFIG_DIR = config.devenv.root + "/apps/reporter";
     FEEDS_CONFIG_DIR = config.devenv.root + "/config";
     REPORTER_SECRET_KEY_FILE_PATH = config.devenv.root + "/nix/test-environments/test-keys";
-    LD_LIBRARY_PATH = lib.makeLibraryPath self'.legacyPackages.commonLibDeps;
```

Comment:
You remove it from here but add in `nix/modules/blocksense/backends/process-compose.nix`

Why is this needed?

Reply:
because this caused problems with dev-shell:
```
8.13.0/lib/libcurl.so.4: undefined symbol: libssh2_session_callback_set2
```